.happy/terraform/envs/prod/main.tf

```diff
@@ -13,7 +13,7 @@ module stack {
   frontend_url                 = "https://cellxgene.cziscience.com"
   backend_url                  = "https://api.cellxgene.cziscience.com"
   stack_prefix                 = ""
-  batch_container_memory_limit = 230000
+  batch_container_memory_limit = 250000
```

Comment:
I would crank it up to 300GB (300000). It's a limit so no need to be shy I think.

Reply:
```suggestion
  batch_container_memory_limit = 300000
```